pkg/cli/subcommands/clusterchecks/command.go

```diff
@@ -103,6 +106,7 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 				fx.Supply(cliParams),
 				fx.Supply(bundleParams(globalParams)),
 				core.Bundle(),
+				secretsnoopfx.Module(), // TODO: secret-enabled: is this required ?
```

Comment:
For this one and the above one I think secrets are not required.
There's an option that controls whether the DCA should resolve secrets or if it should be the cluster check runners. If it's configured not to resolve secrets it's not needed, and if it's configured to resolve, I think they should have already been resolved. Maybe add these 2 to your QA instructions in the PR description for container-platform to be 100% sure.